docs/source/technical_tutorials/access/copc_access.ipynb

```diff
@@ -0,0 +1,291 @@
+{
```

Comment:
<p>The text does not match the code. Your code is using maap-py, not earthaccess.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/459/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>